x.toml

```diff
@@ -69,6 +69,19 @@ include-dev = true
 proc-macros-on-target = false
 # Don't set target or host platforms, or omitted packages, for the full set.
 
+[workspace]
+
+# Regex for allowed characters in paths. Regex must have ^ and $ anchors.
+allowed-paths = "^([a-zA-Z0-9._\\-/@:]|-)+$"
```

Comment:
I think we should be restrictive as we possible can here.   Not sure if you already have with the fuzzers.

Also I think it's fine to make this a default and remove from the config file and perhaps (or not?) allow for overrides.

Reply:
Hmm, I'd like to tackle the fuzzer thing at a later point -- I think they should be excluded from content linters but maybe not from any others.